src/elements/boxandwhiskers.js

```diff
@@ -105,6 +105,14 @@ const BoxAndWiskers = Chart.elements.BoxAndWhiskers = ArrayElementBase.extend({
     ctx.moveTo(x0, boxplot.median);
     ctx.lineTo(x0 + width, boxplot.median);
 
+    // Draw the segment line
+    if (vm.segmentColor) {
+      ctx.strokeStyle = vm.segmentColor;
+    }
+    ctx.beginPath();
+    ctx.moveTo(x0, boxplot.segment);
+    ctx.lineTo(x0 + width, boxplot.segment);
```

Comment:
What happens if `boxplot.segment` is `null`? Shouldn't we put this block inside of an `if(boxplot.segment != null){...}` block

Reply:
Nothing appeared when I tried it out, but you're right. I will add this in right away!